main.js

```diff
@@ -0,0 +1,15 @@
+const openMenu = document.querySelector(".hamburger");
+const closeMenu = document.querySelector(".close-menu");
+const mobileMenu = document.querySelector(".mobile-menu ")
+
+openMenu.addEventListener('click', () => {
+    mobileMenu.style.display = 'flex';
+   
+
+  });
+
+  closeMenu.addEventListener('click', () => {
+    mobileMenu.style.display = 'none';
+
+
+ });
```

Comment:
- [ ] When you click on any navigation link the menu suppose to disappear and the correct part of the page is displayed.

![image](https://user-images.githubusercontent.com/68796232/163043757-a5cd8fcb-58c7-4a7a-a3fa-6ef86f49af28.png)

- [ ] When I  click on `contact` on the mobile menu It doesn't take me to the contact form so please fix that.

Reply:
done